### DISCUSSION TITLE
Не работает в браузере Ice Raven (форк Firefox)

### DISCUSSION THREAD

**Discussion by Someone-G** [anticensority/runet-censorship-bypass#257] [Q&A]
Почему-то не работает в браузере Ice Raven 2.20.0 (форк Firefox). Скачал файл xpi, установил, но расширение не работает.
https://github.com/anticensority/runet-censorship-bypass/releases
![Screenshot_20240612-224105](https://github.com/anticensority/runet-censorship-bypass/assets/108881628/bce4df33-66b6-4c30-b764-a3d8ec009105)

**Comment by ilyaigpetrov** [comment]
На Андроиде не поддерживается этот API (`proxy.settings`): https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/proxy/settings. Без него не будет работать.

**Reply by Someone-G** [reply]
Я в этом ничего не понимаю... 
А что же делать? Раньше я ставил ОБР в этот же браузер (в этом же смартфоне) и всё работало.
